src/mediatype/mediatype-text.ts

```diff
@@ -0,0 +1,16 @@
+export const mediatypeText: { [key: string]: any } = {
```

Comment:
Suggestion: A ... design, software maintenance goal would be to have all the mediatype-related information in one place. So that adding a mediatype or changing a mediatype has one single place: source of truth.

You might have a single object with three properties, icon, text, color